textproc/doxygen/Portfile

```diff
@@ -4,8 +4,8 @@ PortSystem              1.0
 PortGroup               cmake 1.0
 
 name                    doxygen
-version                 1.8.13
-revision                2
+version                 1.8.14
+revision                1
```

Comment:
This would be `revision 0`, except that's the default and we don't put a `revision` line in under those circumstances.